packages/botonic-nlp/src/parser/types.ts

```diff
@@ -0,0 +1,8 @@
+export type Augmenter = { [keyword: string]: string[] }
+export type Entity = { text: string; label: string; start: number; end: number }
+export type Sample = { text: string; entities: Entity[] }
```

Comment:
Sample is too general for being in parser/types. Move to a ner.ts file to rename to NerSample

Reply:
It has to be general, a ParsedData will be the object returned after parsing the input data file. 
That's why each sample has to include the text and the corresponding information. Which is, for the moment, the entities found.

Reply:
yes, you're right! leave it where it is